frontend/src/components/MeetingRoom/InputDevices/InputDevices.tsx

```diff
@@ -38,6 +38,7 @@ const InputDevices = ({ handleToggle, customLightBlueColor }: InputDevicesProps)
     })?.deviceId;
     if (audioDeviceId) {
       publisher?.setAudioSource(audioDeviceId);
+      window.localStorage.setItem('audioSource', audioDeviceId);
```

Comment:
We might want to do one of both of two things:
- extract this interaction with localStorage to its own file to track where we interact with localstorage
- use a shared constant for the keys
currently there will be no error if this line was accidentally 
```ts
      window.localStorage.setItem('audioDevice', audioDeviceId);
```